### PR TITLE
Switch to the MIT License

### PR DESCRIPTION
**Pull request by atosatto**
In order to make the role more open to new contributors and users we decided to drop the GPLv2 license in favor of MIT.

@clwells and @tiawaz since your code is also part of this project, please let us know if you have anything against the re-licensing. If we will not hear any feedback from you in 15 days we will assume you're fine with it and proceed merging this PR.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ galaxy_info:
   author: PowerDNS Engineering Team
   description: PowerDNS Recursor Role
   company: PowerDNS.COM BV
-  license: GPLv2
+  license: MIT
```
Comment (Habbie) [inline meta/main.yml]: whitespace? :)

**Comment by tiawaz** [comment]
license change is fine with me

**Comment by clwells** [comment]
Same - license change is fine with me.

**Comment by atosatto** [comment]
Thank you!

**Pull request merged**